### PR TITLE
compatibility with nested_fit output files

### PR DESCRIPTION
**Pull request by martinit18**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes # (issue)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My code is PEP8 compliant (`flake8 anesthetic tests`)
- [x] My code contains compliant docstrings (`pydocstyle --convention=numpy anesthetic`)
- [x] New and existing unit tests pass locally with my changes (`python -m pytest`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have appropriately incremented the [semantic version number](https://semver.org/) in both README.rst and anesthetic/_version.py

### CODE REVIEW

**Comment by martinit18** [comment]
Please find a more complete pool here

**Comment by martinit18** [comment]
Sorry for this back-forward but it is my first real pull request

File: anesthetic/read/nestedfit.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ----------
+    root : str
+        root specify the directory only, no specific roots,
+        The files read files are ``nf_output_points.txt``
+        and ``nf_output_diag.txt``.
+
+    """
+    if root[-1] != '/':
+        root = root + '/'
+    dead_file = root + 'nf_output_points.txt'
```
Comment (williamjameshandley) [inline anesthetic/read/nestedfit.py:21]: This is best achieved with:
```python
dead_file = os.path.join(root, 'nf_output_points.txt')
```
and the same for `data_birth` and `root_getdist`

**Review by williamjameshandley — CHANGES_REQUESTED** [review]
Looking good @martinit18, just one minor code clean-up and you're ready to merge -- accept the collaborator request and you'll then be able to squash and merge (and not need approval with new pushes).

In addition, you should increase the minor version number so that the version reads `2.2.0`, since we're adding functionality (see [semantic versioning](https://semver.org/))

**Comment by lukashergt** [comment]
Nice, I've incremented the version to 2.2.0 to reflect that we now have nested_fit compatibility :)

**Comment by martinit18** [comment]
I changed the code accordingly to your suggestions.

**Review by williamjameshandley — APPROVED** [review]
Many thanks @martinit18. Please press 'squash and merge'.

**Pull request merged**